README.md

```diff
@@ -134,6 +134,12 @@ Our boolean model is defined in `boolean_model.py`.
 
 ### Vectorial Model
 
+A vectorial model is a statistical model that takes into account the frequency of words in the collections documents and in the query.
+The idea is to compare vectors that represent the query and the documents in the collection and to choose the queries that are closest.
+In this project, there has been no selection of documents returned by the vectorial model. This means that even though the order of the output is probably better that with the boolean model, the documents in the output are the same.
```

Comment:
I disagree. If you read the result section you will see there is not the same number of documents returned.